keycloak.tf

```diff
@@ -59,7 +59,9 @@ resource "aws_ecs_task_definition" "keycloak-ecs-taskdef" {
       "environment": [
         {"name":"KEYCLOAK_ADMIN", "value":"${var.kc_username}"},
         {"name":"KC_DB", "value":"mariadb"},
-        {"name":"KC_DB_URL", "value":"jdbc:mysql://${aws_db_instance.keycloak-database-engine.endpoint}/${var.db_name}?autoReconnect=true"},
+        {"name":"KC_DB_URL_HOST", "value":"${aws_db_instance.keycloak-database-engine.endpoint}"},
+        {"name":"KC_DB_URL_DATABASE", "value":"${var.db_name}"},
+        {"name":"KC_DB_URL_PROPERTIES", "value":"?autoReconnect=true"},
```

Comment:
question: double-checking, these still have `_URL_` in their names, rather than `KC_DB_PROPERTIES`?

Reply:
Yes Paul, the names are correct.